opentracing/utils.go

```diff
@@ -7,12 +7,13 @@ import (
 )
 
 const (
-	// OpenTracingContextIDHTTPHeaderPrefix precedes the opentracing-related
-	// ContextID HTTP headers.
-	OpenTracingContextIDHTTPHeaderPrefix = "Open-Tracing-Context-Id-"
-	// OpenTracingTagsHTTPHeaderPrefix precedes the opentracing-related
-	// trace-tags HTTP headers.
-	OpenTracingTagsHTTPHeaderPrefix = "Open-Tracing-Trace-Tags-"
+	// ContextIDHTTPHeaderPrefix precedes the opentracing-related ContextID HTTP
+	// headers.
+	ContextIDHTTPHeaderPrefix = "Open-Tracing-Context-Id-"
```

Comment:
bikeshed: prefer OpenTracing to Open-Tracing. there's no Open org that we're a part of, unlike google cloud which have Cloud-Foo and Cloud-Bar